static/img/rss_podcasts.svg

```diff
@@ -0,0 +1,88 @@
+         id="Path"
+         stroke="#222326"
+         fill="#000000" />
+      <text
+         xml:space="preserve"
+         style="font-style:normal;font-weight:normal;font-size:12px;line-height:1.25;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none"
+         x="16.05298"
+         y="23.726023"
+         id="text5810"><tspan
+           sodipodi:role="line"
+           id="tspan5808"
+           x="16.05298"
+           y="23.726023"
+           style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:12px;font-family:System-ui;-inkscape-font-specification:'System-ui Bold';fill:#ffffff">OUR RSS FEED</tspan></text>
```

Comment:
D'oh, I found one more place to change `OUR RSS FEED` to `thenet.lol RSS feed` :)